doc/doxygen/headers/simplex.h

```diff
@@ -85,4 +105,11 @@
  * documentation of the enumeration of the numbering of their vertices and
  * lines are given in the right columns.
  *
+ * @section simplex_reference_example_simplex Example: simplex mesh
+ *
+ * @include step_3_simplex.cc
+ *
+ * @section simplex_reference_example_mixed Example: mixed mesh
+ *
+ * @dontinclude step_3_mixed.cc
```

Comment:
```suggestion
```

I'm not sure what your original intent here but it probably was not to include `step_3_simplex.cc` twice in this file. Should this be something else?